ai21_tokenizer/jamba_instruct_tokenizer.py

```diff
@@ -0,0 +1,76 @@
+from __future__ import annotations
+
+import os
+import tempfile
+from pathlib import Path
+from typing import Union, List, cast, Optional
+
+from tokenizers import Tokenizer
+from tokenizers import Tokenizer as TokenizerType
+
+from ai21_tokenizer import BaseTokenizer
+from ai21_tokenizer.utils import PathLike
+
+_TOKENIZER_FILE = "tokenizer.json"
+_DEFAULT_MODEL_CACHE_DIR = Path(tempfile.gettempdir()) / "tokenizer_cache"
+
+
+class JambaInstructTokenizer(BaseTokenizer):
+    _tokenizer: TokenizerType
+
+    def __init__(
+        self,
+        model_path: str,
+        cache_dir: Optional[PathLike] = _DEFAULT_MODEL_CACHE_DIR,
```

Comment:
Probably best this cache dir would include the jamba as a namespace (this way you can have different tokenizer caches)

Reply:
Here do:
`cache_dir = None`
in the first line of the init do `cache_dir = cacher_dir or _DEFAULT_MODEL_CACHE_DIR`

You got me wondering if you have a bug

Reply:
Done. Thanks